Jenkinsfile

```diff
@@ -17,54 +17,37 @@
 @Library('github.com/fabric8io/fabric8-pipeline-library@master')
 def utils = new io.fabric8.Utils()
 def flow = new io.fabric8.Fabric8Commands()
-dockerTemplate{
-   clientsNode {
-    ws{
-      checkout scm
 
-      if (utils.isCI()){
-        def snapshotImageName = "fabric8/maven-builder:SNAPSHOT-${env.BRANCH_NAME}-${env.BUILD_NUMBER}"
-        container('docker'){
-          stage('build snapshot image'){
-            sh "docker build -t ${snapshotImageName} ."
-          }
-          stage('push snapshot image'){
-            sh "docker push ${snapshotImageName}"
-          }
-        }
-        stage('notify'){
-            def pr = env.CHANGE_ID
-            if (!pr){
-                error "no pull request number found so cannot comment on PR"
-            }
-            def message = "snapshot maven-builder image is available for testing.  `docker pull ${snapshotImageName}`"
-            container('docker'){
-                flow.addCommentToPullRequest(message, pr, 'fabric8io-images/maven-builder')
-            }
-        }
-      } else if (utils.isCD()){
-        
-        def v = getNewVersion{}
-        // stage('tag'){
-        //   container('clients'){
-        //     gitTag{
-        //       releaseVersion = v
-        //       skipVersionPrefix = true
-        //     }
-        //   }
-        // }
+dockerTemplate {
```

Comment:
why change existing 2 spaces of indentation? I don't think adding 2 more spaces improves the readability :)
Lets be consistent with the convention used everywhere.

Reply:
@sthaha  Updated the patch to stick with 2 spaces everywhere.